attest/key_windows.go

```diff
@@ -168,19 +156,19 @@ func (k *key20) Quote(t *TPM, nonce []byte, alg HashAlg) (*Quote, error) {
 	return quote20(tpm, tpmKeyHnd, alg.goTPMAlg(), nonce)
 }
 
-// Close frees any resources associated with the key.
-func (k *key20) Close(tpm *TPM) error {
+func (k *key20) close(tpm *platformTPM) error {
 	return closeNCryptObject(k.hnd)
 }
 
 // Delete permenantly removes the key from the system. This method
 // invalidates Key and any further method invocations are invalid.
-func (k *key20) Delete(tpm *TPM) error {
+//
+// TODO(ericchiang): this isn't exported. remove the method or do we need it?
+func (k *key20) Delete(tpm *platformTPM) error {
```

Comment:
@twitchy-jsonp fyi

Reply:
I added it in for a one-time cleanup, and it only makes sense on Windows. We can remove it.